src/transformers/models/graphormer/modeling_graphormer.py

```diff
@@ -816,8 +816,8 @@ def forward(
         out_degree: torch.LongTensor,
         spatial_pos: torch.LongTensor,
         attn_edge_type: torch.LongTensor,
-        perturb=None,
-        masked_tokens=None,
+        perturb: Optional[torch.FloatTensor] = None,
+        masked_tokens: None = None,
```

Comment:
For `perturb` I took what [microsoft/graphormer](https://github.com/microsoft/Graphormer/blob/b8ef0317f45714ab41d6089b5f8d7fc48ed45242/graphormer/tasks/graph_prediction.py#L314) uses.

`masked_tokens` on the other hand, raises a `NotImplemented` whenever it has something that is not None

Reply:
Haa, that's funny. Congrats on our first correct type hint of an explicit `None`!